hypervisor/qemu/qemu_amd64.go

```diff
@@ -60,8 +60,13 @@ func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
 		"-realtime", "mlock=off", "-no-user-config", "-nodefaults", "-no-hpet",
 		"-rtc", "base=utc,clock=vm,driftfix=slew", "-no-reboot", "-display", "none", "-boot", "strict=on",
 		"-m", memParams, "-smp", cpuParams)
-
-	if boot.BootToBeTemplate || boot.BootFromTemplate {
+	if boot.EnableVhostUser {
```

Comment:
@gao-feng  I'm not sure. And I am not even sure about whether hugepage is enabled when the memory is saved on hugetlbfs instead of tmpfs when templating.